vowpalwabbit/v_array.h

```diff
@@ -66,6 +66,14 @@ struct v_array<T, typename std::enable_if<std::is_trivially_copyable<T>::value>:
     memset(_end, 0, (end_array - _end) * sizeof(T));
   }
 
+  // This will move all elements after idx by width positions and reallocate the underlying buffer if needed.
+  // The size must be updated prior to calling this
```

Comment:
Does the comment: `The size must be updated prior to calling this` correspond to updating `_end` by `1` during insert?

Reply:
Yes